src/services/Payments.php

```diff
@@ -122,7 +122,7 @@ public function captureFromGateway(Transaction $transaction): CaptureResponse
 		$authorizedAmount     = (float)$transaction->paymentAmount;
 
 		// Convert amount
-		$amount = $order->getOutstandingBalance()  * $transaction->paymentRate;
+		$amount = $transaction->paymentAmount;
```

Comment:
Look good to me. Maybe we should align the comment to reflect the change? Maybe we could do the same for the comment on line 100 while we're at it?